media-gfx/zbar/files/zbar-0.23.92-py311-set-size.patch

```diff
@@ -0,0 +1,30 @@
+Upstream-PR: https://github.com/mchehab/zbar/pull/231
+
+diff --git a/python/enum.c b/python/enum.c
```

Comment:
I assume you didn't use .patch on the PR URL (which would keep metadata) because it doesn't cleanly apply?

Reply:
I downloaded https://github.com/mchehab/zbar/pull/231/commits/e47f6d04c5ddf9d7ea5227b17eb47d392e9f0fa6.patch but it failed to apply bacuse of https://github.com/mchehab/zbar/commit/5b3c33d5c87f58633b550d592f6eb8e4fd580b1e which was commited between 0.23.92 and master. I manually corrected spaces in line 14 and 26 in this patch, because it seemed to be easier than cherry-picking, resolving the conflict and extracting the patch through git.